src/xrParticles/particle_actions_collection.cpp

```diff
@@ -1622,7 +1622,7 @@ extern void noise3Init();
 #if defined(XR_ARCHITECTURE_X86) || defined(XR_ARCHITECTURE_X64)
 #include <xmmintrin.h>
 #elif defined(XR_ARCHITECTURE_ARM) || defined(XR_ARCHITECTURE_ARM64)
-#include "Externals/sse2neon/sse2neon.h"
+#include "sse2neon.h"
```

Comment:
Should not be global include, should be included like this:
```suggestion
#include "sse2neon/sse2neon.h"
```